contao/controllers/BackendInstall.php

```diff
@@ -731,16 +728,9 @@ protected function enableSafeMode()
 
 		if (!\Config::get('coreOnlyMode'))
 		{
-			global $kernel;
-
-			$modules = array();
-
-			foreach ($kernel->getContaoBundles() as $bundle)
-			{
-				$modules[] = $bundle->getName();
-			}
+			$modules = \ModuleLoader::getActive();
 
-			if (count(array_diff($modules, array('ContaoCoreBundle', 'calendar', 'comments', 'devtools', 'faq', 'listing', 'news', 'newsletter', 'repository'))) > 0)
+			if (count(array_diff($modules, array('core', 'calendar', 'comments', 'faq', 'listing', 'news', 'newsletter'))) > 0)
```

Comment:
This is wrong. We need to use the new bundle names here since they have all been converted.